site/content/en/docs/formats/mars.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 'MARS'
+linkTitle: 'MARS'
+description: ''
+weight: 14
+---
+
+## Format specification
+
+MARS is a dataset for and motion analysis and person identification task,
+and this dataset it's extension of Market-1501 dataset format.
+MARS dataset is available for downloading
+[here](http://zheng-lab.cecs.anu.edu.au/Project/project_mars.html)
+
+Supported types of annotations:
+- `Label`
+
+Required attributes:
+- `is_distractors` (bool): True when image with distractors,
```

Comment:
"When" implies a dependent clause, which needs a verb - but there is no verb here.